src/root.zig

```diff
@@ -417,6 +417,17 @@ pub fn Ymlz(comptime Destination: type) type {
             return str;
         }
 
+        fn getExpressionValueWithTrim(self: *Self, expression: Expression) []const u8 {
+            _ = self;
+
+            // Trim spaces from value
+            return std.mem.trim(u8, switch (expression.value) {
+                .Simple => expression.value.Simple,
+                .KV => expression.value.KV.value,
+                else => @panic("Not implemeted for " ++ @typeName(@TypeOf(expression.value))),
+            }, " ");
```

Comment:
Yep, this works for me, but I think something like this would be a bit cleaner 🫶 

```suggestion
            const value = self.getExpressionValue(expression);
            return std.mem.eql(u8, value, " ");
```

Reply:
Certainly! The inline I used is a little funky :sweat_smile: I shall update as we speak!

Reply:
Huzzah! I made the following adjustments per your feedback:
```
        fn getExpressionValueWithTrim(self: *Self, expression: Expression) []const u8 {
            return std.mem.trim(u8, self.getExpressionValue(expression), " ");
        }
```


Let me know if that seems nonsensical and I can adjust as needed :smile:

Reply:
Looks great, thanks again!